src/sources/mongodb_metrics/types.rs

```diff
@@ -323,10 +323,11 @@ pub struct CommandServerStatusWiredTigerBlockManager {
 pub struct CommandServerStatusWiredTigerCache {
     #[serde(rename = "bytes currently in the cache")]
     pub bytes_total: i64,
-    #[serde(rename = "bytes read into cache")]
-    pub bytes_read_into: i64,
-    #[serde(rename = "bytes written from cache")]
-    pub bytes_written_from: i64,
+    // Unused
```

Comment:
It does seem likely that we meant to publish these as metrics, but also no one seems to be asking for them so I suppose we can drop.